Shared/View/VoteView.swift

```diff
@@ -88,7 +88,34 @@ struct VoteView: View {
             } else {
                 Text("Loading...")
             }
-        }.navigationTitle("Vote").padding()
+            Spacer()
+            HStack {
+                Spacer()
+                Button(action: {
+                        self.presentationMode.wrappedValue.dismiss()
+                    },
+                    label: {
+                        Text("Done")
+                    })
+            }
+        }
+        .navigationTitle("Vote")
+        .padding()
+        .toolbar {
+            Button(action: {
+                Task {
+                    do {
+                        if let activityID = activityViewModel.activity?.id {
+                            _ = try await ActivityViewModel.deleteActivity(activityID)
```

Comment:
You might want to incorporate backend changes introduced in https://github.com/Fun-Gen/Fun-Gen/pull/46 and make UI test delete the activity.